doc/src/vtr/server_mode/index.rst

```diff
@@ -0,0 +1,203 @@
+.. _server_mode:
+
+Server Mode
+================
+
+VTR provides the ability to run in server mode using the following command-line arguments.
+
+.. code-block:: none
+
+  --server --port_num 60555
+
+Server mode may only accept a single client application connection and respond to two types of requests: **get critical path report** and **highlight selected critical path elements**.
+
+Communication telegram
+-------------------------
+
+Telegram consists of two parts: a fixed-size **telegram header** and a variable-size **telegram body**.
+The telegram header contains helper information required to properly extract the telegram body sequence from the data flow.
+
+.. _fig_comm_telegram_structure:
+
+.. figure:: comm_telegram_structure.*
+
+    Communication telegram structure.
+
+    
+.. _fig_comm_telegram_body_structure:
+
+.. figure:: comm_telegram_body_structure.*
+
+    Communication telegram body structure.
+
+    **telegram body** is flat **JSON** structure.
```

Comment:
flat -> a flat

Reply:
fixed https://github.com/verilog-to-routing/vtr-verilog-to-routing/pull/2560/commits/73fc03cdcdacceba3ceb55f74c7fe38be83da32d